src/idsk/components/subscription-form/subscription-form.js

```diff
@@ -0,0 +1,37 @@
+import '../../../govuk/vendor/polyfills/Function/prototype/bind'
+import '../../../govuk/vendor/polyfills/Event' // addEventListener and event.target normaliziation
+import { nodeListForEach } from '../../../../package/govuk/common'
+
+function SubscriptionForm ($module) {
+  this.$module = $module
+}
+
+SubscriptionForm.prototype.init = function () {
+  // Check for module
+  var $module = this.$module
+  if (!$module) {
+    return
+  }
+
+  // button to toggle content
+  var $forms = $module.querySelectorAll('.example-action form')
```

Comment:
toto nemoze byt takto ak je tato classa vlozena cez params: .example-action

Reply:
Ja som na to šiel myšlienkou, že toto niekto nemusí chcieť.
Keď niekto bude robiť vlastnú implementáciu a nebude chcieť využívať našu funkciu, ktorá stav zmení okamžite aj keď sa nič nespraví, tak si iba cez params dá triedu, podla ktorej bude pridávať vlastný event listener a tento bude odignorovaný.

Lebo tak, keď bude niekto robiť to posielanie, tak to môže chcieť spraviť po svojom a až keď sa tie veci vykonajú a príde naspäť odpoveď, sa zmení stav komponentu na poslaný.

Alebo toto nemám riešiť vôbec alebo navrhuješ nejaké lepšie riešenie? 
Alebo to poistím ešte tak, že ak žiadna param.class nebude, tak dá aspoň túto? Alebo to premenovať aj na param.targetClass, nech to je viac zjavne?

Reply:
Okej, chapem co si tym chcel. Ak to chceme takto, tak by som dal minimalne iny nazov lebo v js komponente mat ".example-action" vyznie ako keby to niekto tam zabudol. Dal by som to skor nieco ako ". idsk-subscription-form__event-handler" pricom "event-handler" je nazov co handluje (v sucasnom komponente je len jeden event-hadler, ale ak by ich bolo viac, mali by sme ich vediet rozlisovat ak by sme ich chceli tiez takto "personalizovat") plus description v yaml by obsahoval tuto poznamku (akurat som si vsimol ze teraz pri classes aj zly popis "Class name for component banner" :D ). Alebo ano dalsia moznost ako si pisal dat to ako samostatny params a nemiesat to do classes a s dostatocnym popisom.

Reply:
Okay, tak to upravim tak. Ale ak je nejaké lepšie riešenie alebo je niekde inde riešený tento problém, tak to kľudne spravím podľa toho.